src/main/java/org/breedinginsight/daos/TraitDAO.java

```diff
@@ -259,7 +258,6 @@ public TraitDAO(Configuration config, DSLContext dsl, BrAPIProvider brAPIProvide
                     .build();
             BrApiTrait brApiTrait = BrApiTrait.builder()
                     .traitName(trait.getTraitName())
-                    .traitDescription(trait.getDescription())
```

Comment:
Similar comment as for the v1 observationvariables endpoint regarding autopopulation. Although I'm wondering what would happen in the case of an update. When we update in our other brapi endpoints we retrieve what's on in the brapi service then update fields and put that. Would we just overwrite those fields with autopopulated data?

Reply:
I would think that we would adjust the value of the autopopulated fields as the fields that autopopulate them change.

Reply:
Ya, makes sense.